test/test_quantization.py

```diff
@@ -775,6 +775,43 @@ def test_minmax_observer(self, qdtype, qscheme, reduce_range):
         self.assertEqual(qparams[1].item(), ref_zero_point)
         self.assertAlmostEqual(qparams[0].item(), ref_scale, delta=1e-5)
 
+    @given(qdtype=st.sampled_from((torch.qint8, torch.quint8)),
+           qscheme=st.sampled_from((torch.per_channel_affine, torch.per_channel_symmetric)))
+    def test_per_channel_minmax_observer(self, qdtype, qscheme):
+        myobs = PerChannelMinMaxObserver(ch_axis=1, dtype=qdtype, qscheme=qscheme)
+        x = torch.tensor([[1.0, 2.0, 2.0, 3.0, 4.0, 6.0], [-4.0, 5.0, 5.0, 6.0, 7.0, 8.0]])
```

Comment:
Please add a test for 4D tensors, I found a bug in the original implementation as it worked only for 2d